src/DatasourceWithAsyncBackend.ts

```diff
@@ -85,7 +93,7 @@ export class DatasourceWithAsyncBackend<
     let allData: DataFrame[] = [];
 
     return getRequestLooper(
-      { ...request, targets: [target], requestId: `aws_ts_${this.requestCounter++}` },
+      { ...request, targets: [target], requestId: `${this.requestIdPrefix}_${this.requestCounter++}` },
```

Comment:
Instead of using the instance id as the prefix, timestream uses a global requestCounter to avoid conflicting requestIDs from different timestream datassources, I don't really have a strong opinion on how we distinguish them

Reply:
one potential issue with using a global here is different versions of this module being loaded and the issue could still be present.

a (very) edge case scenario: athena and redshift both use `DatasourceWithAsyncBackend` with a global counter, but they end up being bundled with different versions of the `@grafana/async-data-query` package. a dashboard with an athena and redshift panel would load different `DatasourceWithAsyncBackend` modules and they both generate the same `requestId`.

Reply:
That makes sense!